Server/controllers/user.controller.js

```diff
@@ -1,17 +1,66 @@
-"use strict";
+const datalayerFactory = require('../datalayer/factory.datalayer');
```

Comment:
Please remove if we don't need `datalayerFactory `

Reply:
Removed